app/components/clock/Clock.jsx

```diff
@@ -11,19 +11,40 @@ class Clock extends React.Component {
     };
 
     state = {
-        dateNow: this._getCurrentDate()
+        hours:      this._getCurrentHours(),
```

Comment:
Suggest to use only `date` property in `state`, and extract hours/minutes/seconds from it directly in `render` method.
Something like this:

```javascript
state = {
  date: new Date()
};

_getTime() {
  const { date } = this.state;
  const hours = date.getHours();
  ...
  return { hours, minutes, seconds };
}
...
render() {
  const { hours, minutes, seconds } = this._getTime();
  ...
}
```